packages/notifier/test/test-stored-subscription.js

```diff
@@ -17,7 +17,7 @@ import {
   makeFakeMarshaller,
 } from '../tools/testSupports.js';
 
-import '../src/types-ambient.js';
+import '../src/types.js';
```

Comment:
I'm surprised that this works. `types.js` is now a module, so types defined there are not ambient, right?

Reply:
it's a noop. I'll remove it if I push again